AcadeMeData/views.py

```diff
@@ -1,6 +1,8 @@
 from django.shortcuts import render, redirect
 from django.contrib import messages
-from .forms import UserRegistrationForm
+from .forms import UserRegistrationForm, MessageForm
+from .models import Messages
+from django.contrib.auth.decorators import login_required
 
```

Comment:
login required handles the auth of the user 
if youre logged out you will be redirected to login page